### ISSUE TITLE
feat(gauss_plume): amortized NPE predictor for the emission-rate posterior (Step 5)

### ISSUE THREAD

**Issue by jejjohnson** [jejjohnson/plumax#87]
## Problem / Request
Tier I is the only tier cheap enough to skip the emulator, which makes it the right place to build the first Step 5 amortized predictor: a neural posterior estimator trained on simulated `(θ, y)` pairs that returns `p(Q, x₀, H | y, met)` in one forward pass. The roadmap lists `gauss_plume.predictor` ☐, with the context-conditioning layer (FiLM / hypernet) coming from `pyrox.nn`. No amortized inference exists anywhere in the repo.

## User Story
> As a user screening thousands of overpasses, I want a trained predictor that returns a calibrated posterior on `Q` in milliseconds, validated against the NUTS posterior, so I can run the population layer at scale.

## Proposed API
```python
# src/plumax/gauss_plume/predictor.py
class PlumePredictor(eqx.Module):
    flow: eqx.Module                    # conditional normalising flow over (log Q, x0, y0, H)
    conditioner: eqx.Module             # FiLM on (u, θ_wind, stability, background) context
    def sample(self, y: jax.Array, context: jax.Array, key, n: int) -> jax.Array: ...
    def log_prob(self, theta, y, context) -> jax.Array: ...

def simulate_training_set(key, *, n: int, prior, receptors, noise: Noise) -> tuple[jax.Array, jax.Array, jax.Array]: ...
def train_predictor(model, data, *, steps: int, lr: float) -> PlumePredictor: ...   # pipekit-train / optax
```

## Implementation Steps
- [ ] Add `src/plumax/gauss_plume/predictor.py` (equinox flow; `pyrox` as an optional `predictor` extra)
- [ ] `simulate_training_set` draws from the same prior and `Noise` the NUTS model uses (likelihoods module)
- [ ] Training loop through `pipekit_train` if available, plain optax otherwise
- [ ] Export a `PerEventPosterior`-compatible sample object for the Tier V catalog

## Definition of Done
- [ ] On 200 held-out synthetic overpasses the predictor's posterior mean/std on `Q` match NUTS within Monte-Carlo error
- [ ] SBC over 1000 simulations: rank histograms uniform, stratified by stability class
- [ ] `make test && make lint && make typecheck` green (training test `@pytest.mark.slow`)

## Testing
- [ ] `tests/gauss_plume/test_predictor.py::test_matches_nuts_posterior` (`slow`)
- [ ] `tests/gauss_plume/test_predictor.py::test_sbc_uniform` (`slow`)

## Relationships
- Parent (theme epic): #70
- Blocked by: likelihoods module (shared loss)
- Related: Tier V per-event payload v2
